lib/si-cli/src/cmd/update.rs

```diff
@@ -114,21 +114,23 @@ impl AppState {
         let current_containers = self.container_engine().get_container_details().await?;
 
         let mut containers = Vec::new();
-        let latest_containers: Vec<LatestContainer> = req.json().await?;
-        'outer: for latest in latest_containers {
-            for current in &current_containers {
-                if current.image != format!("{}/{}", latest.namespace, latest.repository) {
-                    continue;
-                }
+        if !current_containers.is_empty() {
```

Comment:
We don't want to run through ANY updates if there are no containers currently installed